### PR TITLE
Extracting non-palette color vars to a new file

### PR DESCRIPTION
**Pull request by latin-panda**
# Description
Extracting color variables to another file, the ones that are not part of the palette.
The variables in private-vars.less are still accessible anywhere that imports variable.less, so the original idea doesn't work. However having them in 2 files might help devs to understand the right way of using them

https://github.com/medic/cht-core/issues/7862

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs:

* CHT Core: CHT_CORE_COMPOSE_URL
* CouchDB Single: COUCH_SINGLE_COMPOSE_URL
* CouchDB Cluster: COUCH_CLUSTER_COMPOSE_URL
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @m5r, can you please review? 
Perhaps you know a better alternative to this solution 
 :) Thanks!

File: webapp/src/css/private-vars.less [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/**
+* WARNING:
+* Don't import this file. This is only for "variable.less" usage.
+* Don't use these variables. The variable names aren't descriptive enough.
+* Instead, use the variables from "variable.less".
+*/
```
Comment (m5r) [inline webapp/src/css/private-vars.less:6]: I couldn't find a solution to reduce Less variables' scope so this is the best we can do for now :+1:

File: webapp/src/css/private-vars.less [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+@white: #FFFFFF;
```
Comment (m5r) [inline webapp/src/css/private-vars.less:7]: This variable is still used in https://github.com/medic/cht-core/blob/7862-private-color-variables/webapp/src/css/content-list.less#L225 but I don't know if that's intended

File: webapp/src/css/private-vars.less [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+@gray-ultra-light: #F2F2F2;
+@gray-light: #E0E0E0;
+@gray-medium: #A0A0A0;
+@gray-dark: #777777;
+@gray-ultra-dark: #333333;
+@black: #000000; ///aaaaaaa
```
Comment (m5r) [inline webapp/src/css/private-vars.less:13]: Is the comment still needed?

**Review by m5r — APPROVED** [review]
I'm sorry I don't have an alternative to propose here but it's still an improvement over what we had before, thank you Jenni!  
I've left a few small comments but I'll approve so as to not block you until I come back on monday

File: webapp/src/css/private-vars.less (hunk 1 above)
Comment (latin-panda) [reply webapp/src/css/private-vars.less:6]: Cool, thanks for checking!

File: webapp/src/css/private-vars.less (hunk 2 above)
Comment (latin-panda) [reply webapp/src/css/private-vars.less:7]: Yes, this is intended, that piece changed with the [bulk delete work](https://github.com/medic/cht-core/pull/7846/files#diff-7531bbe609e35f2936eac69d48cd591e18105b79dce25dc64de3c5c4833c1418L224) it was my attempt to reduce code conflicts :)

File: webapp/src/css/private-vars.less (hunk 3 above)
Comment (latin-panda) [reply webapp/src/css/private-vars.less:13]: hahaha sorry for the weird comment, I forgot to remove it. Thanks for catching it

**Pull request merged**